runtime/frequency/src/lib.rs

```diff
@@ -751,6 +751,7 @@ impl GetStableWeight<RuntimeCall, Weight> for CapacityEligibleCalls {
 			RuntimeCall::StatefulStorage(StatefulStorageCall::upsert_page { payload, ..}) => Some(capacity_stable_weights::SubstrateWeight::<Runtime>::upsert_page(payload.len() as u32)),
 			RuntimeCall::StatefulStorage(StatefulStorageCall::delete_page { .. }) => Some(capacity_stable_weights::SubstrateWeight::<Runtime>::delete_page()),
 			RuntimeCall::StatefulStorage(StatefulStorageCall::apply_item_actions_with_signature { payload, ..}) => Some(capacity_stable_weights::SubstrateWeight::<Runtime>::apply_item_actions_with_signature(StatefulStorage::sum_add_actions_bytes(&payload.actions))),
+			RuntimeCall::StatefulStorage(StatefulStorageCall::apply_item_actions_with_signature_v2 { payload, ..}) => Some(capacity_stable_weights::SubstrateWeight::<Runtime>::apply_item_actions_with_signature(StatefulStorage::sum_add_actions_bytes(&payload.actions))),
```

Comment:
@enddynayn Is this enough to allow capacity transactions or I need to do more?

Reply:
Do we need to update `frequency-tx-payment/src/capacity_stable_weights.rs`?
If so, do we need to run benchmarks to update the weights?
Should this be:
```suggestion
			RuntimeCall::StatefulStorage(StatefulStorageCall::apply_item_actions_with_signature_v2 { payload, ..}) => Some(capacity_stable_weights::SubstrateWeight::<Runtime>::apply_item_actions_with_signature_v2(StatefulStorage::sum_add_actions_bytes(&payload.actions))),
```

Reply:
Just noticed the comment in the issue that weights haven't changed, so it seems like this comment can be ignored.

Reply:
Well I could copy and paste all the weights and benchmarks but not sure it worths the effort since we are going to remove the old extrinsic in near future

Reply:
I think I see, if the weights are the same, I agree, no need to copy and paste a duplicate set.
The only other thing I was thinking is that the integration tests do test for every capacity eligible transaction.
Do we want to test the v1 and v2 concurrently?

Reply:
So thanks for reminding to add capacity integration test for this new extrinsic. I'll add that